frontend/src/navbar.tsx

```diff
@@ -29,12 +29,16 @@ export function Navbar(props: BoxProps) {
    const setPullFilter = useSetFilter();
    const {toggleColorMode} = useColorMode();
    const [showCryo, setShowCryo] = useBoolUrlState('cryo', false);
+   const [showBlocked, setShowBlocked] = useBoolUrlState('external_block', false);
    const hideBelowMedium = ['none', 'none', 'block'];
    const hideBelowLarge = ['none', 'none', 'none', 'block'];
 
    const toggleShowCryo = useCallback(() => setShowCryo(!showCryo), [showCryo]);
    useEffect(() => setPullFilter('cryo', showCryo ? null : isPullCryo), [showCryo]);
 
+   const toggleShowBlocked = useCallback(() => setShowBlocked(!showBlocked), [showBlocked]);
```

Comment:
I am not sure about using just `Blocked` as part of these names since we have other things besides `External Blocked` that can be blocked like `CI Blocked`, `Deploy Blocked`, and `Dev Blocked`. 

What about using `ExtBlocked`?

Reply:
I agree that this could have a better name convention, I like **ExtBlocked** ✅